samples/app-anonymous-users/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-app-anonymous-users-nodejs",
+    "source": "officeDev",
+    "title": "Anonymous User Support",
+    "shortDescription": "This sample app shows anonymous users support in teams meeting apps.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/app-anonymous-users/nodejs",
+    "longDescription": [
+      "This sample app shows anonymous users support in teams meeting apps."
```

Comment:
This sample app shows anonymous users support in `Teams` meeting apps. It also supports Azure AD and Facebook authentications.

Reply:
Fixed